docs/language/operators/from.md

```diff
@@ -54,20 +66,105 @@ from ... | switch color (
 ) | ...
 ```
 
-The output of a fork consists of multiple legs that must be merged.
-If the downstream operator expects a single input, then the output legs are
-merged with an automatically inserted [combine operator](combine.md).
```

Comment:
_This_ was wholly redundant with what's in the `fork` doc, so I _did_ drop this, since we already link to the `fork` doc.